### PR TITLE
Unit testing/mobile front/hugo

### PR DESCRIPTION
**Pull request by hugolopespinto**
.Add News tests in the front mobile
pages: item opinions, favorites

image

### CODE REVIEW

**Review by CedricCORGE — APPROVED** [review]

File: Application/mobile/app_code/lib/pages/article/favorite/favorite_page.dart [hunk 1]
```diff
@@ -3,8 +3,11 @@ import 'package:flutter/material.dart';
 import 'favorite_state.dart';
 
 class FavoritePage extends StatefulWidget {
+  final List<dynamic> favorites;
```
Comment (31Nathan) [inline Application/mobile/app_code/lib/pages/article/favorite/favorite_page.dart:6]: prefix them with test to ensure the understanding of it

File: Application/mobile/app_code/lib/pages/article/favorite/favorite_state.dart [hunk 2]
```diff
@@ -45,6 +49,7 @@ class FavoriteSate extends State<FavoritePage> {
       if (response.statusCode == 200) {
         setState(() {
           favorites = jsonDecode(response.body)['favorites'];
+          print(favorites);
```
Comment (31Nathan) [inline Application/mobile/app_code/lib/pages/article/favorite/favorite_state.dart:52]: Remove debug

File: Application/mobile/app_code/lib/pages/opinion/opinion_page.dart [hunk 3]
```diff
@@ -4,12 +4,15 @@ import 'opinion_state.dart';
 
 class OpinionPage extends StatefulWidget {
   final int itemId;
+  final List<dynamic> opinions;
```
Comment (31Nathan) [inline Application/mobile/app_code/lib/pages/opinion/opinion_page.dart:7]: Same here, prefix them with tests

**Review by 31Nathan — CHANGES_REQUESTED** [review]
Small changes

**Review by 31Nathan — APPROVED** [review]

**Pull request merged**